keras_hub/src/models/efficientnet/efficientnet_presets.py

```diff
@@ -0,0 +1,29 @@
+"""EfficientNet preset configurations."""
+
+backbone_presets = {
+    "efficientnet_b0_ra_imagenet": {
+        "metadata": {
+            "description": (
+                "EfficientNet B0 model pre-trained on the ImageNet 1k dataset "
+                "with RandAugment recipe."
```

Comment:
what is the random augment recipie?

Reply:
Here's the [paper](https://arxiv.org/abs/1909.13719)... should we link it?